pkg/cmd/attestation/verification/sigstore_integration_test.go

```diff
@@ -43,31 +43,13 @@ func TestLiveSigstoreVerifier(t *testing.T) {
 	})
 
 	t.Run("with missing verification material", func(t *testing.T) {
-		attestations := getAttestationsFor(t, "../test/data/github_provenance_demo-0.0.12-py3-none-any-bundle-missing-verification-material.jsonl")
-		require.NotNil(t, attestations)
-
-		verifier := NewLiveSigstoreVerifier(SigstoreConfig{
-			Logger: io.NewTestHandler(),
-		})
-
-		res := verifier.Verify(attestations, publicGoodPolicy(t))
-		require.Error(t, res.Error)
-		require.ErrorContains(t, res.Error, "failed to get bundle verification content")
-		require.Nil(t, res.VerifyResults)
+		_, err := GetLocalAttestations("../test/data/github_provenance_demo-0.0.12-py3-none-any-bundle-missing-verification-material.jsonl")
+		require.ErrorContains(t, err, "missing verification material")
 	})
 
 	t.Run("with missing verification certificate", func(t *testing.T) {
-		attestations := getAttestationsFor(t, "../test/data/github_provenance_demo-0.0.12-py3-none-any-bundle-missing-cert.jsonl")
-		require.NotNil(t, attestations)
-
-		verifier := NewLiveSigstoreVerifier(SigstoreConfig{
-			Logger: io.NewTestHandler(),
-		})
-
-		res := verifier.Verify(attestations, publicGoodPolicy(t))
-		require.Error(t, res.Error)
-		require.ErrorContains(t, res.Error, "leaf cert not found")
-		require.Nil(t, res.VerifyResults)
+		_, err := GetLocalAttestations("../test/data/github_provenance_demo-0.0.12-py3-none-any-bundle-missing-cert.jsonl")
```

Comment:
Do we still need to call `verifier.Verify` or is that now handled by the updated sigstore-go function?

Reply:
The new version of sigstore-go includes some [additional validation](https://github.com/sigstore/sigstore-go/pull/285) during processing of the bundle, which causes these two bundles (`github_provenance_demo-0.0.12-py3-none-any-bundle-missing-cert.jsonl` and `github_provenance_demo-0.0.12-py3-none-any-bundle-missing-verification-material.jsonl`) to fail to load in `GetLocalAttestations` (which is also called by the helper `getAttestationsFor` used in the previous version of these tests).

Essentially, this means we fail faster here. Instead of failing during verification, it fails during bundle loading/parsing. The result for the user should be roughly the same, but probably with a different error message.

Reply:
That sounds great.